src/rad/inspector/wxfbadvprops.cpp

```diff
@@ -578,10 +594,33 @@ wxVariant wxFBBitmapProperty::ChildChanged(wxVariant& thisValue, const int child
 
                     break;
                 }
-                // 'Load From Art Provider'
-                case 4:
+				// 'Load From XRC'
+				case 4:
+				{
+					if (prevSrc != 4)
+					{
+						for (unsigned int i = 1; i<count; i++)
+						{
+							wxPGProperty *p = Item(i);
+							if (p)
+							{
+								wxLogDebug(wxT("wxFBBP::ChildChanged: Removing:%s"), p->GetLabel().c_str());
+								GetGrid()->DeleteProperty(p);
+							}
+						}
+						bp->AppendChild(bp->CreatePropertyXrcName());
+					}
+
+					if (childVals.GetCount() == 2)
+						newVal = childVals.Item(0) + wxT("; ") + childVals.Item(1);
+					else if (childVals.GetCount() > 1)
+						newVal = childVals.Item(0) + wxT("; ");
+					break;
+				}
+				// 'Load From Art Provider'
+                case 5:
                 {
-					if( prevSrc != 4 )
+					if( prevSrc != 5 )
```

Comment:
Hm ... we should do something about these magic numbers.

Reply:
This is one of the smaller problems, a much bigger problem is not only the usage of the display strings as data storage but also directly writing them into the project file. Currently its not possible to localize wxFB without creating localized project files that can only be processed with a wxFB using the same locale. Plus there are too many strings marked for translation, but this is not that hard to fix.

Reply:
I don't think wxFB will ever by localized (and IMHO it also never should be).